infrastructure/smart-contracts/contracts/RefereeCalculations.sol

```diff
@@ -36,17 +36,33 @@ contract RefereeCalculations is Initializable, AccessControlUpgradeable {
      */
     function calculateChallengeEmissionAndTier(
         uint256 totalSupply,
-        uint256 maxSupply
+        uint256 maxSupply,
+        uint256 challengeStart,
+        uint256 challengeEnd
     ) public pure returns (uint256, uint256) {
         require(maxSupply > totalSupply, "5");
+        require(challengeEnd >= challengeStart, "7"); // Ensure that end time is not before start time
 
         uint256 tier = Math.log2(maxSupply / (maxSupply - totalSupply)); // calculate which tier we are in starting from 0
         require(tier <= 23, "6");
 
+        // Calculate the emission tier
         uint256 emissionTier = maxSupply / (2 ** (tier + 1)); // equal to the amount of tokens that are emitted during this tier
 
+        // TODO: not sure if this is the correct way to calculate the emissions
+        uint256 emissionsPerHour = emissionTier / 17520; // 17520 is the number of hours in a year        
+        
```

Comment:
Please confirm this logic/calculation for per hour emissions is correct.

Reply:
@XDapps Is that not the exact same way the Referee currently calculates the amount / challenge ?
Currently the Referee will assume a challenge happens every hour as per the[ GitBook docs ](https://xai-foundation.gitbook.io/xai-network/about-xai/sentry-nodes-explained/accruing-node-rewards)